src/App.js

```diff
@@ -1,24 +1,18 @@
 import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import { BrowserRouter, Route, Switch } from 'react-router-dom';
+import AddView from './Components/AddView/addView';
+import ListView from './Components/ListView/listView';
+import './index';
 
 function App() {
   return (
     <div className="App">
```

Comment:
Is it necessary to use a `<div>` here when the code is already nested within the `<BrowserRouter>`?